packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -33,24 +33,25 @@ export { extraClasses as cx };
       aria-hidden={false}
       aria-label="success"
     />
-  </div>
 
-  <div class="mr-auto">{message}</div>
+    {message}
```

Comment:
Nit: wrap this in a `<p class="truncate">` to adhere to the design guidelines that:

> Toasts can be up to 480px wide. **No text wrapping**